
 Makefile                 |   22 +++++++++++++---------
 scripts/kconfig/Makefile |    8 --------
 2 files changed, 13 insertions(+), 17 deletions(-)

diff -Nurp linux-3.0-rc6/Makefile linux-3.0-rc6-mrproper/Makefile
--- linux-3.0-rc6/Makefile	2011-07-08 00:23:36.551892719 +0300
+++ linux-3.0-rc6-mrproper/Makefile	2011-07-08 00:29:19.249251111 +0300
@@ -502,8 +502,9 @@ ifeq ($(KBUILD_EXTMOD),)
 # Carefully list dependencies so we do not try to build scripts twice
 # in parallel
 PHONY += scripts
-scripts: scripts_basic include/config/auto.conf include/config/tristate.conf
-	$(Q)$(MAKE) $(build)=$(@)
+#scripts: scripts_basic include/config/auto.conf include/config/tristate.conf
+#	$(Q)$(MAKE) $(build)=$(@)
+scripts:
 
 # Objects we will link into vmlinux / subdirs we need to visit
 init-y		:= init/
@@ -986,7 +987,8 @@ prepare0: archprepare FORCE
 	$(Q)$(MAKE) $(build)=. missing-syscalls
 
 # All the preparing..
-prepare: prepare0
+#prepare: prepare0
+prepare:
 
 # Generate some files
 # ---------------------------------------------------------------------------
@@ -1158,11 +1160,13 @@ CLEAN_FILES +=	vmlinux System.map \
                 .tmp_kallsyms* .tmp_version .tmp_vmlinux* .tmp_System.map
 
 # Directories & files removed with 'make mrproper'
-MRPROPER_DIRS  += include/config usr/include include/generated          \
-                  arch/*/include/generated
-MRPROPER_FILES += .config .config.old .version .old_version             \
-                  include/linux/version.h                               \
-		  Module.symvers tags TAGS cscope* GPATH GTAGS GRTAGS GSYMS
+#MRPROPER_DIRS  += include/config usr/include include/generated          \
+#                  arch/*/include/generated
+#MRPROPER_FILES += .config .config.old .version .old_version             \
+#                  include/linux/version.h                               \
+#		  Module.symvers tags TAGS cscope* GPATH GTAGS GRTAGS GSYMS
+MRPROPER_DIRS  +=
+MRPROPER_FILES +=
 
 # clean - Delete most, but leave enough to build external modules
 #
@@ -1180,7 +1184,7 @@ clean: archclean
 #
 mrproper: rm-dirs  := $(wildcard $(MRPROPER_DIRS))
 mrproper: rm-files := $(wildcard $(MRPROPER_FILES))
-mrproper-dirs      := $(addprefix _mrproper_,Documentation/DocBook scripts)
+mrproper-dirs      := $(addprefix _mrproper_,Documentation/DocBook)
 
 PHONY += $(mrproper-dirs) mrproper archmrproper
 $(mrproper-dirs):
diff -Nurp linux-3.0-rc6/scripts/kconfig/Makefile linux-3.0-rc6-mrproper/scripts/kconfig/Makefile
--- linux-3.0-rc6/scripts/kconfig/Makefile	2011-07-08 00:23:39.085910043 +0300
+++ linux-3.0-rc6-mrproper/scripts/kconfig/Makefile	2011-07-08 00:27:50.174635275 +0300
@@ -12,26 +12,18 @@ Kconfig := Kconfig
 endif
 
 xconfig: $(obj)/qconf
-	$< $(Kconfig)
 
 gconfig: $(obj)/gconf
-	$< $(Kconfig)
 
 menuconfig: $(obj)/mconf
-	$< $(Kconfig)
 
 config: $(obj)/conf
-	$< --oldaskconfig $(Kconfig)
 
 nconfig: $(obj)/nconf
-	$< $(Kconfig)
 
 oldconfig: $(obj)/conf
-	$< --$@ $(Kconfig)
 
 silentoldconfig: $(obj)/conf
-	$(Q)mkdir -p include/generated
-	$< --$@ $(Kconfig)
 
 # if no path is given, then use src directory to find file
 ifdef LSMOD
